lua/wikis/commons/Widget.lua

```diff
@@ -29,6 +29,8 @@ local Widget = Class.new(function(self, props)
 	self.context = {} -- Populated by the parent
 end)
 
+---@alias Renderable string|Html|Widget|number|nil
```

Comment:
do we maybe want `?` instead of `nil`?
we might have nothing passed instead of nil in some places